e2e_tests/support/commands.ts

```diff
@@ -139,9 +139,7 @@ Cypress.Commands.add('addUser', (userName, password, role, force) => {
   if (force === true) {
     cy.get('[type=checkbox]').click()
   }
-  cy.get('[class*=Button]')
-    .contains('Add User')
-    .click()
+  cy.get('[data-testid="Add User"]').click()
```

Comment:
This e2e command was using a generated class from styled components to select a button. When parsing with typescript, for some reason the generated classes were updated so the selector stopped working. I added a data-testid attribute on the button to use as a selector instead.